games/static/games/proto.js

```diff
@@ -7,22 +7,40 @@ class Game {
     this.restartButton = document.querySelector("#restart");
     this.isRunning = false
   }
+  win(board) {
+    return !board.containsAliens()
+  }
+  checkLose(board, gameId) {
```

Comment:
name is confusion - made me feel it should return a boolean
also the function is weird because sometime it returns a value and sometimes it doesnt
when looking into `lose()` we can see it doesnt return anything...
what did you intend to do in this funciton.
align the name with your intention and the rest will follow
and actually